docs/install-node.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# In the ReadTheDocs build, this file is ran by Sphinx' conf.py to install
+# support for Node.js.
+
+set -e
+
+
+if [ "$READTHEDOCS" = 'True' ]; then
```

Comment:
Yup, I just wanted to be sure somebody won't execute this locally and cause some mess on their computer. Leaving out the `if` in the config file would make the config file's logic less clear, I think. So I better made this redundant.